### DISCUSSION TITLE
Sharing pictures with family

### DISCUSSION THREAD

**Discussion by Eirikr70** [immich-app/immich#3886] [Q&A] [answered]
Hello, I have set up my Immich pictures base ans shared it with my familiars. I thought we would share a unique base altogether but I see that each of us has their proper base and can share it with the others. Is their no way to make it one only base without using the same credentials ? Thank you

**Comment by korotovsky** [comment, marked as answer]
Please see my comment here https://github.com/immich-app/immich/discussions/1633#discussioncomment-6834726

**Reply by Eirikr70** [reply]
Thanks, seems to answer my question.

**Reply by korotovsky** [reply]
Hi @Eirikr70,

Here is a PR https://github.com/immich-app/immich/pull/3890, but it is likely to be delayed to merge, because it seems my PR is currently is blocked by other PR. I have no estimate when it will be merged unfortunately.

**Reply by Eirikr70** [reply]
Hello @korotovsky, thanks for the precision. So I'll start with separate bases for each family member, which is still ok although not optimal in my use case. Then I suppose that when I migrate to the new solution, I would have to, in a way, bulk download all the pictures under the new fake user and process the face recognition from scratch ?...
